packages/x-charts/src/internals/plugins/featurePlugins/useChartVisibilityManager/isIdentifierVisible.ts

```diff
@@ -0,0 +1,23 @@
+import type { VisibilityMap } from './useChartVisibilityManager.types';
+
+export const VISIBILITY_SEPARATOR = '-';
+
+export const buildIdentifier = (ids: string | (string | number)[]) => {
```

Comment:
Should we be more explicit in types like that?

```suggestion
export const buildIdentifier = (ids: SeriesId | [SeriesId, ItemId]) => {
```